index.qmd

```diff
@@ -1,55 +1,190 @@
+  </div>
+  Some text about how easy it is to interface with external packages like AbstractGPs. Learn more about modelling [Gaussian Processes](https://turinglang.org/docs/tutorials/15-gaussian-processes/) with Turing.jl.
+</div>
+<div class="example-code">
+```julia
+@model function putting_model(d, n; jitter=1e-4)
+    v ~ Gamma(2, 1)
+    l ~ Gamma(4, 1)
+    f = GP(v * with_lengthscale(SEKernel(), l))
+    f_latent ~ f(d, jitter)
+    binomials = Binomial.(n, logistic.(f_latent))
+    y ~ product_distribution(binomials)
+    return (fx=f(d, jitter), f_latent=f_latent, y=y)
+end
+```
+</div>
+</div>
 
+```{=html}
+<h3 class="pb-3">
+  News
+</h3>
+```
+::: {#news}
 
-::: {.hero-buttons style="position: relative; display: flex; justify-content: center;"}
-[Get Started]({{< meta get-started >}}){.btn-action-primary .btn-action .btn .btn-lg role="button" style="background-color: #073c44; color: white"}
 :::
 
-::: {.g-col-3}
-:::
-:::
-:::
+[See all news &rarr;](news/)
 
-:::
-
-::: {style="position: relative; display: flex; justify-content: center;"}
-::: {.grid}
-::: {.g-col-1}
-:::
-::: {.g-col-8}
-## Community Resources
-
-The Turing.jl [discourse forum](https://discourse.julialang.org/c/domain/probprog) provides support and discussion for all user levels.
-The [Turing.jl documentation]({{< meta get-started >}}) and [official tutorials]({{< meta tutorials-intro >}}) help users learn and use Turing.jl!
-There's also a [tutorial in Bayesian Statistics using Julia and Turing.jl](https://storopoli.io/Bayesian-Julia).
-
-:::
-::: {.g-col-3}
-:::
-:::
-:::
+```{=html}
+<h3 class="pb-3 section-start-space">
+  Learn more
+</h3>
+<div class="list grid quarto-listing-cols-3">
+  <a class="pseudolisting" href="https://discourse.julialang.org/c/domain/probprog/48">
+    <div class="card-title h5">
+      Discourse
+    </div>
+    <p class="quarto-grid-item">
+      Get support from and discuss with other users.
+    </p>
+  </a>
+  <a class="pseudolisting" href="https://turinglang.org/docs/tutorials/docs-00-getting-started/">
+    <div class="card-title h5">
+      Documentation
+    </div>
+    <p class="quarto-grid-item">
+      Familiarize yourself with the Turing.jl API.
+    </p>
+  </a>
+  <a class="pseudolisting" href="https://turinglang.org/docs/tutorials/00-introduction/">
+    <div class="card-title h5">
+      Official tutorials
+    </div>
+    <p class="quarto-grid-item">
+      Learn to tackle specific problems with Turing.jl.
+    </p>
+  </a>
+  <a class="pseudolisting" href="resources/content/inofficial_tutorials.qmd">
+    <div class="card-title h5">
+      Inofficial tutorials
+    </div>
+    <p class="quarto-grid-item">
+      A list of inofficial tutorials using Turing.jl.
+    </p>
+  </a>
+  <a class="pseudolisting" href="resources/content/talks.qmd">
+    <div class="card-title h5">
+      Talks
+    </div>
+    <p class="quarto-grid-item">
+      A list of talks on the Turing.jl ecosystem.
+    </p>
```

Comment:
This is the sort of code I'm talking about – it's repetitive, and that's just the nature of raw HTML. In a framework like Astro you would be able to declare something like `<GridItem link="..." title="...">`.

Reply:
+1 for Astro! It would be a great choice if it possible to integrate the existing docs workflow.

While working on this PR, I was simultaneously working on another "docs-like" page that uses [Astro Starlight](https://starlight.astro.build/). I often felt bad repeating all these HTML chunks over and over and wished I could just wrap them in a component...

I don't think I'm a good web dev, but I think it's a fun creative outlet. If you ever go through with the move to Astro or another proper JS framework, I'd be happy to help in the effort.

Reply:
I probably wasn't clear enough in my original comment, sorry. I think we can still build the docs repo with Quarto and this repo with Astro (or similar) It sounds like a good follow-up stretch goal 😄 I'll open an issue to track that.

Reply:
https://github.com/TuringLang/turinglang.github.io/issues/110

Reply:
Ah, sorry, I think your explanation was clear! I phrased the "integrate the existing docs workflow" poorly. That should have read "integrate with the existing quarto docs workflow", oopsies.